gammapy/spectrum/models.py

```diff
@@ -1063,31 +1065,174 @@ def plot(self, energy_range, ax=None, energy_unit='TeV',
         return ax
 
 
-class AbsorbedSpectralModel(SpectralModel):
+class Absorption(object):
+    """
+    Class dealing with absorption model
+
+    Parameters
+    ----------
+    energy_lo : `~astropy.units.Quantity`
+        Lower bin edges of energy axis
+    energy_hi : `~astropy.units.Quantity`
+        Upper bin edges of energy axis
+    param_lo : `~astropy.units.Quantity`
+        Lower bin edges of parameter axis
+    param_hi : `~astropy.units.Quantity`
+        Upper bin edges of parameter axis
+    data : `~astropy.units.Quantity`
+        Model value
+
+    Examples
+    --------
+    Create and plot EBL absorption model for a redshift of 0.2
+
+    .. create::
```

Comment:
Does this exist / work? I've never seen the `.. create::` Sphinx directive. Usually we use the `.. plot::` Sphinx directive.